owners/mapbiomas-public.jsonnet

```diff
@@ -0,0 +1,11 @@
+
+{
+  "id": "mapbiomas-public",
+  "homeBucket": "projects/mapbiomas-public",
+  "name": "MapBiomas",
+  "description": "MapBiomas is a collaborative network of NGOs, universities, and technology companies focused on developing annual land use and land cover maps for Brazil and other countries using satellite imagery and machine learning. The project promotes transparency, scientific rigor, and open access to environmental data for monitoring land use changes over time.",
```

Comment:
could you also suggest two images with the dimensions matching imageLink and logo in https://github.com/google/earthengine-catalog/blob/main/owners/landandcarbon.jsonnet? 
The small logo image will be shown on each dataset page, while the large imageLink image will represent Mapbiomas in the overall publisher list https://developers.google.com/earth-engine/datasets/publisher